examples/40_paper/2015_neurips_feurer_example.py

```diff
@@ -24,6 +24,7 @@
 ####################################################################################################
 # List of dataset IDs given in the supplementary material of Feurer et al.:
 # https://papers.nips.cc/paper/5872-efficient-and-robust-automated-machine-learning-supplemental.zip
+# fmt: off
```

Comment:
Is this for black? What would happen if you turn it off?

Reply:
Yes it is for Black. It would put each element of the list in a separate row.